### PR TITLE
Creating a UI to change the properties in the Config

### LINKED ISSUE #59: Creates an UI to edit the config file
Create an UI to easily edit the config file

### PR DESCRIPTION
**Pull request by PrincessVika**
Early version of the UI:
- no validations for now (so if you change something to something that doesnt make sense, the .py wont work! e.g. "Base Ball: asd")
- no video capture : it is implemented and you could have a video capture, but for now it does nothing, so useless.
- you can't start the .py from the UI, so you still need to use the Config.py to start the program. The UI just changes the values in the TOML for an easy user experience.

### CODE REVIEW

**Comment by pifopi** [comment]
Linked to issue #59

File: AutoMaxLair.runtimeconfig.dev.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "runtimeOptions": {
+    "additionalProbingPaths": [
+      "C:\\Users\\vikto\\.dotnet\\store\\|arch|\\|tfm|",
```
Comment (pifopi) [inline AutoMaxLair.runtimeconfig.dev.json]: can you give relative path instead ?

File: config.txt [hunk 2]
```diff
@@ -0,0 +1,29 @@
+Cresselia
```
Comment (pifopi) [inline config.txt]: why is this file needed ? it's seems like a copy from the .toml file

**Comment by pifopi** [comment]
I don't see the C# code. Did you forgot to add it ? Also, adding myself a memo to show you how to revert some changed on git

**Review by pifopi — APPROVED** [review]
Well done ! It's fine for me ! I gave you some ideas to further improve it but i'm writing them here as well (+ some more as I thought about it) :

- Check if config.toml exist. If it doesn't, create it by reading config.sample.toml
- Give some hints on what the user should input for every value (even if it's less needed than on the config file itself because the UI greatly improve the UX)
- Some kind of data validation (warn the user he inputed a string instead of an int, an int being too large, etc)
- Have a windows that you can pop up to select the path to tesseract instead of relying only on the string
- Maybe read the config.sample.toml while saving the config file to force some values such as the detected sentences for every language. That would ensure people update those (but maybe that issue should be fixed by splitting the config files ?)
- Either force the window to not be resizable (or to support resizing)

File: AutoMaxLairUI - Code/AutoMaxLair.csproj.user [hunk 3]
```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (pifopi) [inline AutoMaxLairUI - Code/AutoMaxLair.csproj.user]: that file should be deleted from the depot

File: AutoMaxLairUI - Code/Properties/PublishProfiles/FolderProfile.pubxml [hunk 4]
```diff
@@ -0,0 +1,18 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (pifopi) [inline AutoMaxLairUI - Code/Properties/PublishProfiles/FolderProfile.pubxml]: that file should be deleted from the depot

File: AutoMaxLairUI - Code/Properties/PublishProfiles/FolderProfile.pubxml.user [hunk 5]
```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (pifopi) [inline AutoMaxLairUI - Code/Properties/PublishProfiles/FolderProfile.pubxml.user]: that file should be deleted from the depot

File: config.txt [hunk 6]
```diff
@@ -0,0 +1,29 @@
+Landorus-incarnate
```
Comment (pifopi) [inline config.txt]: that file should be deleted from the depot

File: AutoMaxLairUI - Code/Form1.cs [hunk 7]
```diff
@@ -0,0 +1,614 @@
...
+            t["discord"]["WEBHOOK_TOKEN"] = boxWebhookToken.Text;
+            t["discord"]["USER_ID"] = boxUserID.Text;
+            t["discord"]["USER_SHORT_NAME"] = boxPingName.Text;
+            t["discord"]["UPDATE_LEVELS"] = boxPingSettings.Text;
+
+            // Game Language Setting
+            t["language"]["LANGUAGE"] = boxGameLanguage.Text;
+
+            // Pokémon Data Settings
+            t["pokemon_data_paths"]["Boss_Pokemon"] = "data/boss_pokemon.json";
```
Comment (pifopi) [inline AutoMaxLairUI - Code/Form1.cs]: for all values below, I'd read directly from the config.sample.toml. That way, if there is a change to the config file, no need to update the gui

**Review by pifopi — CHANGES_REQUESTED** [review]

File: AutoMaxLairUI - Code/AutoMaxLair.csproj.user (hunk 3 above)
Comment (PrincessVika) [reply AutoMaxLairUI - Code/AutoMaxLair.csproj.user]: deleted!

File: AutoMaxLairUI - Code/Properties/PublishProfiles/FolderProfile.pubxml (hunk 4 above)
Comment (PrincessVika) [reply AutoMaxLairUI - Code/Properties/PublishProfiles/FolderProfile.pubxml]: deleted!

File: AutoMaxLairUI - Code/Properties/PublishProfiles/FolderProfile.pubxml.user (hunk 5 above)
Comment (PrincessVika) [reply AutoMaxLairUI - Code/Properties/PublishProfiles/FolderProfile.pubxml.user]: deleted!

File: config.txt (hunk 6 above)
Comment (PrincessVika) [reply config.txt]: deleted!

File: AutoMaxLairUI - Code/Form1.cs (hunk 7 above)
Comment (PrincessVika) [reply AutoMaxLairUI - Code/Form1.cs]: uses the sample to get the language values!

**Review by pifopi — APPROVED** [review]
Great job ! Thanks a lot for your help
